Final project proposal/Proposal_Yixuan&Hang4.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Applying AI to identify spatial clustering of pedestrian activity and street environment features"
+author: Yixuan Zhao & Hang Tian

```

Comment:
nice to have a team project.  Be sure to commit the work you do separately under you own names so I can see what's been written by each of you.